dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -67,6 +67,16 @@ TaskFunction createGalleryBackButtonMemoryTest() {
   );
 }
 
+TaskFunction createPlatformServicesStartupTest() {
+  return new StartupTest(
+      '${flutterDirectory.path}/examples/platform_services',
```

Comment:
My understanding was that we are interested in testing correctness of platform services. If so, then `StartupTest` may not be a suitable way of testing them. Consider writing a driver test that exercises the platform services, and then a devicelab task that runs the driver test.